index.html

```diff
@@ -1 +1,32 @@
-<h1>Hello!</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RecycleMe</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+  <body>
+
+    <div id="login">
+        <form id="login-form" action="">
+            <h3>Please log in or register</h3>
+            Username:<br />
+            <input type="text" name="user-name" value="" />
+            <br />
+            Password:<br />
+            <input type="text" name="password" value="" />
```

Comment:
the new user form will have to include a field for password confirmation. the new_user and login routes are also different, I think it will be easiest to make them different forms